cookbooks/cdo-ruby/attributes/default.rb

```diff
@@ -1,6 +1,6 @@
 default['cdo-ruby'] = {
-  version: '3.0.5',
-  rubygems_version: '3.3.22',
-  bundler_version: '2.3.22',
-  rake_version: '13.0.1'
+  version: '3.3.4',
+  rubygems_version: '3.5.17 ',
```

Comment:
The default version of rubygems with ruby 3.3.4 is actually 3.5.11. I've picked a slightly newer version following a commit of Elijah's: https://github.com/code-dot-org/code-dot-org/commit/bf41ebfcda92dcb5bfc6673ef016e5f9625ae843 where forcing a `gem update --system` was important to things functioning. I'm not sure if its applicable here too, but it was easy to do so 🤷.